client/common.go

```diff
@@ -196,10 +199,12 @@ func (j *JenkinsCore) Request(method, api string, headers map[string]string, pay
 		req      *http.Request
 		response *http.Response
 	)
-
 	if req, err = http.NewRequest(method, fmt.Sprintf("%s%s", j.URL, api), payload); err != nil {
 		return
 	}
+	if Language != "" && Language != "en" {
```

Comment:
`Language != "en"` why?

Reply:
Because the "en" is added to the default value, the unit test will not pass, so it is excluded.

Reply:
`en` is not the default value. The default value is no value.

Reply:
I will take the default value is no value